packages/web-pkg/src/composables/piniaStores/theme.ts

```diff
@@ -94,19 +93,29 @@ export const useThemeStore = defineStore('theme', () => {
   const initializeThemes = (themeConfig: WebThemeConfigType) => {
     availableThemes.value = themeConfig.themes.map((theme) => merge(themeConfig.defaults, theme))
 
-    if (unref(currentThemeName) === null) {
-      currentThemeName.value = unref(availableThemes).find((t) => t.isDark === unref(isDark)).name
-    }
-
+    const firstLightTheme = unref(availableThemes).find((theme) => !theme.isDark)
+    const firstDarkTheme = unref(availableThemes).find((theme) => theme.isDark)
     setAndApplyTheme(
-      unref(availableThemes).find((t) => t.name === unref(currentThemeName)) ||
-        availableThemes.value[0]
+      unref(availableThemes).find((t) => t.name === unref(currentLocalStorageThemeName)) ||
+        (unref(isDark) ? firstDarkTheme : firstLightTheme),
```

Comment:
Do we need a happy path? If the localstorage is empty, dark is applied but not existing ?